src/js/grapl-cdk/Dockerfile

```diff
@@ -3,16 +3,18 @@
 # of other services/builds.
 
 FROM node:alpine3.10 AS grapl-cdk-build
-WORKDIR /home/grapl
-COPY . .
```

Comment:
this early COPY invalidates the rest of the cache. no improvement for CI but definite improvements for local build